NeoML/src/Dnn/Layers/GELULayer.cpp

```diff
@@ -46,52 +66,147 @@ void CGELULayer::Reshape()
 
 	outputDescs.SetSize( 1 );
 	outputDescs[0] = inputDesc;
+
+	if( IsBackwardPerformed() ) {
+		erfMemoization = CDnnBlob::CreateBlob( MathEngine(), CT_Float, inputDesc );
```

Comment:
There's more effective way for this.
Return `TInputBlobs | TOutputBlobs` in `CGELULayer::BlobsForBackward` and you may use `outputBlobs` in `BackwardOnce`

Reply:
It is not an output that is being saved in `erfMemoization`.

Reply:
my bad, resolved